raiden/network/transport/matrix/transport.py

```diff
@@ -650,36 +650,47 @@ def _initialize_first_sync(self) -> None:
         self._client._sync()
         self._client.set_sync_limit(prev_sync_limit)
 
-    def _initialize_broadcast_rooms(self) -> None:
-        msg = "To join the broadcast rooms the Matrix client to be properly authenticated."
-        assert self._user_id, msg
-
-        for suffix in self._config.broadcast_rooms:
-            room_name = make_room_alias(self.chain_id, suffix)
-            broadcast_room_alias = f"#{room_name}:{self._server_name}"
-            self.log.debug("Joining broadcast room", broadcast_room_alias=broadcast_room_alias)
-            self._broadcast_rooms[room_name] = join_broadcast_room(
-                client=self._client, broadcast_room_alias=broadcast_room_alias
-            )
-
     def _initialize_room_inventory(self) -> None:
         msg = "The rooms can only be inventoried after the first sync."
         assert self._client.sync_token, msg
 
         self.log.debug("Inventory rooms", rooms=self._client.rooms)
 
         for room in self._client.rooms.values():
-            if self._is_broadcast_room(room):
-                # Broadcast rooms are write-only, don't listen on them
-                continue
```

Comment:
@ulope I saw in your PR #5343 that you moved this check here. Not sure if this change and the one in #5343 have different expectations?

Reply:
I didn't change this, just how it is implemented.

The loop bellow will check if the room is a broadcast room, if it is, the room is added to the `_broadcast_rooms` and the *listener is not installed*, if it is not a broadcast room, then the listener is installed.

The logic is not the clearest, but this way we can avoid re-joining the broadcast room on every restart, which takes a little while.